src/app/scripts/cac/map/cac-map-control.js

```diff
@@ -425,7 +439,7 @@ CAC.Map.Control = (function ($, Handlebars, cartodb, L, turf, _, UserPreferences
                     }).on('dragend', function(e) {
                         dragging = false;
                         var coords = e.target.getLatLng();
-                        addWaypoint(itinerary, [coords.lng, coords.lat],
+                        addWaypoint(itinerary, [coords.lat, coords.lng],
```

Comment:
Maybe add a note in the comment on `addWaypoint` making it clear that the waypoint argument is [lat, lon] and the startDragPoint argument is [lon, lat].